collectors/osidb_scheduler/collectors.py

```diff
@@ -0,0 +1,69 @@
+"""
+OSIDB Collectors
+"""
+from celery.utils.log import get_task_logger
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import OuterRef, Q, Subquery
+from django.db.models.functions import Cast
+
+from collectors.framework.models import Collector
+from osidb.mixins import Alert
+
+logger = get_task_logger(__name__)
+
+
+class StaleAlertCollector(Collector):
+    """
+    Stale Alert Collector
+    """
+
+    def collect(self):
```

Comment:
I would use:

```python
from osidb.core import set_user_acls
from django.conf import settings
set_user_acls(settings.ALL_GROUPS)
```
To ensure ACLs are always properly bypassed before doing any collection so nothing is missed.